deploy/rbd/kubernetes/csi-rbdplugin.yaml

```diff
@@ -29,7 +29,7 @@ spec:
           securityContext:
             privileged: true
             allowPrivilegeEscalation: true
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.4.0
+          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.1
```

Comment:
this commit misses the update for the NFS-provisioner

Reply:
@nixpanic done.. thanks.. ptal.